README.md

```diff
@@ -11,17 +11,47 @@ This plugin will send metrics regarding route performance on every request to th
 For the `prefix`, please the name of the service that you are integrating with (neato-service, cool-api, etc)
 
 ```
-$ npm install -S @goodwaygroup/lib-hapi-trace-headers
+$ npm install -S @goodwaygroup/lib-hapi-good-tracer
 ```
 
 In your `index.js` for the Hapi server, register the plugin:
 
 ```js
 await server.register({
-    plugin: require('@goodwaygroup/lib-hapi-trace-headers'),
+    plugin: require('@goodwaygroup/lib-hapi-good-tracer'),
     options: {
         traceUUIDHeader: 'x-custom-trace-uuid', // optional defaults to 'x-gg-trace-uuid'
-        traceSeqIDHeader: 'x-custom-trace-seqid' // optional defaults to 'x-gg-trace-seqid'
+        traceSeqIDHeader: 'x-custom-trace-seqid', // optional defaults to 'x-gg-trace-seqid'
+        cache: {
+            ttl: 10 // 10 seconds
```

Comment:
I liked the pattern that you had with specifying the defaults. I think that would work here too.
I think 10 sec is short for an example as I can see someone copying the code block and not changing it or not understanding why they would need to change it.
```suggestion
            ttl: 60 // optional defaults to 120 seconds
```

Reply:
Updated